### ISSUE TITLE
Upgrade from 1.43.3 to 1.43.4 on UDM

### ISSUE THREAD

**Issue by o42o** [nextdns/nextdns#943]
**Context**
 - CLI Version: 1.43.3
 - Platform: UDM / Unifo OS v3.2.12 / Network 8.1.127

I run the command `nextdns upgrade` and it doesn't upgrade from 1.43.3 to 1.43.4. It seems to detect that there is an upgrade available, but the message at the end says that the most recent version, 1.43.3, is installed. I've tried uninstalling NextDNS and reinstalling it, but I get the same result.

```
root@UDM:~# nextdns upgrade
INFO: OS: debian
INFO: GOARCH: arm64
INFO: GOOS: linux
INFO: NEXTDNS_BIN: /usr/bin/nextdns
INFO: INSTALL_RELEASE: 1.43.4
INFO: Upgrading NextDNS...
Hit:1 https://deb.debian.org/debian bullseye InRelease
Get:2 https://deb.debian.org/debian bullseye-updates InRelease [44.1 kB]       
Get:3 https://security.debian.org/debian-security bullseye-security InRelease [48.4 kB]
Get:4 https://deb.debian.org/debian bullseye-backports InRelease [49.0 kB]     
Hit:5 https://repo.nextdns.io/deb stable InRelease                             
Hit:6 https://apt.artifacts.ui.com bullseye InRelease                        
Get:7 https://security.debian.org/debian-security bullseye-security/main arm64 Packages [269 kB]
Get:8 https://deb.debian.org/debian bullseye-backports/main arm64 Packages [402 kB]
Fetched 812 kB in 1s (603 kB/s)
Reading package lists... Done
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
nextdns is already the newest version (1.43.3).
0 upgraded, 0 newly installed, 0 to remove and 49 not upgraded.
```

Next, the command `nextdns version ` returns
```
root@UDM:~# nextdns version
nextdns version 1.43.3
```

**Comment by photomme** [comment]
Same here:
`root@UDM-SE:~# nextdns upgrade
INFO: OS: debian
INFO: GOARCH: arm64
INFO: GOOS: linux
INFO: NEXTDNS_BIN: /usr/bin/nextdns
INFO: INSTALL_RELEASE: 1.43.4
INFO: Upgrading NextDNS...
Hit:1 https://security.debian.org/debian-security bullseye-security InRelease
Hit:2 https://deb.debian.org/debian bullseye InRelease                         
Hit:3 https://deb.debian.org/debian bullseye-updates InRelease                 
Hit:4 https://deb.debian.org/debian bullseye-backports InRelease               
Hit:5 https://repo.nextdns.io/deb stable InRelease                             
Hit:6 https://apt.artifacts.ui.com bullseye InRelease                          
Hit:7 https://apt-beta.artifacts.ui.com bullseye InRelease                     
Hit:8 https://apt-release-candidate.artifacts.ui.com bullseye InRelease
Reading package lists... Done
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
nextdns is already the newest version (1.43.3).
The following packages were automatically installed and are no longer required:
  node20.10 unifi-pion-gw
Use 'apt autoremove' to remove them.
0 upgraded, 0 newly installed, 0 to remove and 22 not upgraded.
`

**Comment by o42o** [comment]
This seems to have been resolved with version 1.43.5.

```
root@UDM:~# nextdns upgrade
INFO: OS: debian
INFO: GOARCH: arm64
INFO: GOOS: linux
INFO: NEXTDNS_BIN: /usr/bin/nextdns
INFO: INSTALL_RELEASE: 1.43.5
INFO: Upgrading NextDNS...
Hit:1 https://deb.debian.org/debian bullseye InRelease
Get:2 https://deb.debian.org/debian bullseye-updates InRelease [44.1 kB]       
Get:3 https://deb.debian.org/debian bullseye-backports InRelease [49.0 kB]     
Get:4 https://security.debian.org/debian-security bullseye-security InRelease [48.4 kB]
Get:5 https://repo.nextdns.io/deb stable InRelease [8490 B]                    
Hit:6 https://apt.artifacts.ui.com bullseye InRelease                          
Get:7 https://repo.nextdns.io/deb stable/main arm64 Packages [426 B]
Fetched 150 kB in 1s (114 kB/s) 
Reading package lists... Done
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following packages will be upgraded:
  nextdns
1 upgraded, 0 newly installed, 0 to remove and 49 not upgraded.
Need to get 2967 kB of archives.
After this operation, 0 B of additional disk space will be used.
Get:1 https://repo.nextdns.io/deb stable/main arm64 nextdns arm64 1.43.5 [2967 kB]
Fetched 2967 kB in 1s (3113 kB/s)  
(Reading database ... 49612 files and directories currently installed.)
Preparing to unpack .../nextdns_1.43.5_arm64.deb ...
Unpacking nextdns (1.43.5) over (1.43.3) ...
Setting up nextdns (1.43.5) ...
NextDNS installed and started using ubios init

root@UDM:~# nextdns version
nextdns version 1.43.5
```
